specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchservice.json

```diff
@@ -5784,6 +5784,82 @@
       },
       "description": "Ranking function based on the Okapi BM25 similarity algorithm. BM25 is a TF-IDF-like algorithm that includes length normalization (controlled by the 'b' parameter) as well as term frequency saturation (controlled by the 'k1' parameter)."
     },
+    "Semantic": {
+      "properties": {
+        "semanticConfigurations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SemanticConfiguration"
+          },
+          "description": "The semantic configurations for the index."
+        }
+      },
+      "externalDocs": {
+        "url": "https://docs.microsoft.com/en-us/azure/search/semantic-search-overview"
+      },
+      "description": "Defines parameters for a search index that influence semantic capabilities."
+    },
+    "SemanticConfiguration": {
+      "properties": {
+        "name": {
+          "externalDocs": {
+            "url": "https://docs.microsoft.com/rest/api/searchservice/Naming-rules"
+          },
+          "type": "string",
+          "description": "The name of the semantic configuration.",
+          "x-nullable": false
+        },
+        "prioritizedFields": {
+          "$ref": "#/definitions/PrioritizedFields",
+          "x-nullable": false,
+          "description": "Describe how the service should process document fields in the context of semantic capabilities.",
```

Comment:
How about: "Specify multiple strings fields, in priority order, that provide the input for semantic ranking, captions, highlights, and answers".

Reply:
I like that phrasing but it doesn't call out that users are specifying three lists.

How about something like: "Describes the title, content, and keyword fields to be used for semantic ranking, captions, highlights, and answers."

Reply:
That's clear to me.